iOS-TvShows/iOS-TvShows/Common/UI/Bundles/BundleImage.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  BundleImage.swift
+//  iOS-TvShows
+//
+//  Created by Filip Ban on 09.11.2021..
+//
+
+enum BundleImage: String {
```

Comment:
Rather than having manual entries for names we have a tool which will autogenerate this from XCAssets folder ;) https://github.com/infinum/redbreast